functions/src/definitions/checkerHandlers.js

```diff
@@ -212,17 +212,25 @@ async function onFactCheckerYes(voteRequestPath, from, platform = "whatsapp") {
       }
       break
   }
-  await voteRequestSnap.ref.update({
+  const updateObj = {
     hasAgreed: true,
     acceptedTimestamp: Timestamp.fromDate(new Date()),
     sentMessageId: res.data.messages[0].id,
-  })
-  await sleep(3000)
-  await sendL1CategorisationMessage(
-    voteRequestSnap,
-    messageRef,
-    res.data.messages[0].id
-  )
+  }
+  if (message.machineCategory === "info") {
+    // handle case where machine has predicted info, so no need to go into L1 categorisation, but still need to go into L2 voting
+    updateObj.triggerL2Vote = true
+    updateObj.triggerL2Others = false
+    await sleep(1000)
```

Comment:
what is this sleep 1000 and 3000 for?

Reply:
Because i realise that sometimes even after we fire the whatsapp API to send image messages to the checker, there is a delay. This results in the categorisation/voting prompts "Which of these categories best describes this message" / "On a scale of 0-5 how true is this message" coming before the image message that they should check. So I added these sleeps to slow down the categorisation/voting prompts so they are likelier to come after the message to check is received by the user.

For the first case (autocategorised as info), I put a shorter sleep cos the message is fired by the onVoteRequest function, which in itself takes some time to trigger.